src/view/screens/Home.tsx

```diff
@@ -198,12 +208,11 @@ function HomeScreenReady({
     return <CustomFeedEmptyState />
   }, [])
 
-  const [homeFeed, ...customFeeds] = allFeeds
   const homeFeedParams = React.useMemo<FeedParams>(() => {
     return {
       mergeFeedEnabled: Boolean(preferences.feedViewPrefs.lab_mergeFeedEnabled),
       mergeFeedSources: preferences.feedViewPrefs.lab_mergeFeedEnabled
-        ? preferences.feeds.saved
+        ? preferences.savedFeeds.map(f => f.value)
```

Comment:
Seems like we'd want to exclude `following` from this.

Reply:
Good call. Now that I'm thinking about it, should we exclude pinned feeds too? The current functionality does not, but does it make sense to sample in Discover if you also have it pinned?

Reply:
I wouldn't mess with that here yet, but IMO the whole "mergefeed" feature needs a rethink pass, it's kind of stuck as a v0.1 experiment